build-system/tasks/karma.conf.js

```diff
@@ -69,6 +78,19 @@ module.exports = {
     transform: [['babelify', {caller: {name: 'test'}, global: true}]],
     // Prevent "cannot find module" errors on Travis. See #14166.
     bundleDelay: isTravisBuild() ? 5000 : 1200,
+
+    persistentCache: browserifyPersistFs(
+      './build/.karma-cache',
+      {
+        deps: createHash(fs.readFileSync('./yarn.lock')),
+        build: globby.sync('build-system/**/*.js').map((f) => {
```

Comment:
According to [this line](https://github.com/martinheidegger/browserify-persist-fs/blob/4f543cf87955016db4f8f2cfca6dc5f7fabdf979/test/cache.js#L22) in the `browserify-persist-fs` source, should this field be called `source` instead of `build`?